modules/mlst/functions.nf

```diff
@@ -0,0 +1,68 @@
+//
+//  Utility functions used in nf-core DSL2 module files
+//
+
+//
```

Comment:
This file will need to be replaced with [this one](https://github.com/nf-core/tools/blob/dev/nf_core/module-template/modules/functions.nf) as a result of the recent changes we have made to be able to record multiple software versions per module.